tests/test_integrated_channels/test_utils.py

```diff
@@ -276,3 +276,36 @@ def test_is_url_valid(self):
     @ddt.unpack
     def test_get_courserun_duration_in_hours(self, course_run, expected_duration_days):
         assert utils.get_courserun_duration_in_hours(course_run) == expected_duration_days
+
+    def test_resize_item_collections(self):
```

Comment:
tests of the resize logic, we indirectly tested it in the past as part of other tests but never directly tested the resize/truncate logic independently.